### DISCUSSION TITLE
Authenticate your SCIM client

### DISCUSSION THREAD

**Discussion by edroszcz** [community/community#106673] [Enterprise]
Hi!

I am testing SAML SSO and SCIM user provisioning for our Enterprise Cloud instance using Entra ID as the IdP.

So far I am testing to provision and offboard my own user account.

When I assign my Entra ID user to the Entra ID Enterprise Application the provisioning is successful and I receive the invitation email from GitHub to my work email address. I can then create a GitHub user account and when I sign in with the account I am part of the organization.

The problem I am having is when I remove my Entra ID user from the Entra ID Enterprise Application. When the Entra ID provisioning cycle is running I get the following error message when it tries to disable my account:

**Unable to deprovision the admin authenticated to perform the SCIM actions. Authenticate your SCIM client with a different account and try again.**

When I configured SSO and SCIM I used my personal Entra ID account but a GitHub "local" account that is a organization admin.

I guess I somehow managed to do something so that the SCIM integration is dependent on my personal Entra ID account.

But I can't figure out how to fix that and associate it with another, more generic, service account.

Do you guys have any ideas :) ?

// Erik

**Comment by hpsin** [comment]
That's an error being sent back from GitHub to Entra to ensure you don't accidentally break provisioning. When you set up SCIM you minted a PAT with your GitHub account, which is what's being used by Entra to authenticate the request. SCIM has to prove it has the rights to add people to your org. There's safeguards in place, though, to ensure that you don't use that token to remove the user who is powering that token. If your GitHub account was removed from the organization, that account's PAT wouldn't be allowed to add people to the org, which would break it. 

If you want to use a different account to power SCIM, you can create a new account in both Entra and GitHub and make that GitHub account an admin of your GitHub org. Then, use a PAT from that account to power SCIM.
